madgiraffestudios/src/components/Booking/Booking.js

```diff
@@ -37,7 +37,7 @@ const [message, setMessage] = useState ('');
       name: fullName,
       phone: phone, 
       from_name: email,
-      to_name: 'Madgiraffestudios@gmail.com',
+      to_name: 'MadGiraffeSTuDios',
```

Comment:
how would they know the email domain

Reply:
this change was for kae to view not the client it's for the email template

Reply:
ooh i see your right

Reply:
Addressed